src/views/select-the-year.njk

```diff
@@ -0,0 +1,57 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk'     import govukBackLink %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/radios/macro.njk' import govukRadios %}
+
+{% block pageTitle %}
+  Select your penalty details
+{% endblock %}
+
+{% block backLink %}
+  {{ 
+    govukBackLink({
+      text: "Back",
+      href: navigation.back.href
+    }) 
+  }}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-width-container">
+    <main role="main" class="govuk-main-wrapper">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <form action="?{{ navigation.actions.continue }}" method="post" id="penaltyForm">
+
+            {{ 
+              govukRadios({
+                idPrefix: 'select-penalty',
+                name: 'selectPenalty',
```

Comment:
this could also be `'select-penalty'`

Reply:
The name has to stay as is because that's the key name that gets returned on submit. 'select-penalty' is not a good key name in TS